### DISCUSSION TITLE
Creating 2+ anki cards that share the same output

### DISCUSSION THREAD

**Discussion by Cold-Neighborhood898** [ObsidianToAnki/Obsidian_to_Anki#465] [Q&A]
Hi There

By chance does anyone know how to create two separate Anki cards that share the same output using this plugin?

For example, my obsidian page might look like this
- Question A
- Question B
- Content A

The anki cards generated would be:
Card 1: Question A --> Content A
Card 2: Question B --> Content A

Thank you for any efforts made in trying to help answer this question!

**Comment by envico801** [comment]
Hi @Cold-Neighborhood898, it is possible as long as the questions are different, because the question is like the unique id that the plugin uses to see if it is a duplicate or not.

**The 2 questions can share the same content but it is also necessary that the content is repeated**.

For example:

```text
Question: What colour is the sun?  
Answer: Green
```

```text
Question: How can I jump higher?  
Answer: Green
```

From what I understand what you want to achieve is this:

```text
Question: What colour is the sun?  
Question: How can I jump higher?  
Answer: Green
```

But I don't know if it's possible, you would have to modify the original plugin to achieve this, but that's more work. 😅

**Reply by Cold-Neighborhood898** [reply]
Thank you for the reply, I really appreciate it!

That is what I am trying to achieve. I managed to achieve that result using the frozen fields function of the plugin.
